_posts/2023-07-24-whats-new-with-bzlmod.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: posts
+title: "What's New with Bzlmod?"
+authors:
+ - wyv
+---
+
+Since the official launch of Bzlmod in Bazel 6.0 in December 2022, the team has been hard at work improving Bzlmod in time for enabling it by default in 7.0, slated to be launched by the end of this year. This blog post will give you an idea of what we've done so far, and what changes you can expect in the future.
```

Comment:
"end of this year" --> November?

Reply:
I didn't want to promise this... 5.0 was released in Jan 2022, and 6.0 was released in Dec 2022. So we might slip into Dec 2023 for 7.0.